### DISCUSSION TITLE
failed loading lazyvim.config.keymaps

### DISCUSSION THREAD

**Discussion by Bamii** [LazyVim/LazyVim#722] [Q&A]
![image](https://user-images.githubusercontent.com/16966438/236676504-d44a6548-0d75-4e86-926a-8efec46c3caa.png)
Neovim Version: 0.9.0
Platform: Windows

I just installed Neovim and subsequently, lazy.vim.
No other steps have been taken.

**Comment by folke** [comment]
This is probably because your NEovim install is corrupted. Try re-installing the latest stable/nightly

**Reply by Bamii** [reply]
I initially installed neovim using Chocolatey. I uninstalled the chocolatey version and tried using the zip installer, and it worked!.
I wanted to determine if it was an issue during my first installation, so i tried installing with Chocolatey again... but the same error came up.
Turns out that the chocolatey version has issues with lazyvim. I haven't tested on another windows environment to be sure that its just my system or its a general thing though.
